packages/react-composites/src/composites/CallComposite/utils/Utils.ts

```diff
@@ -289,35 +289,44 @@ export const isDisabled = (option: boolean | { disabled: boolean } | undefined):
   return option.disabled;
 };
 
+/**
+ * @returns Permissions state for the camera.
+ */
+const queryCameraPermissionFromPermissionsAPI = async (): Promise<PermissionState | 'unsupported'> => {
+  try {
+    return (await navigator.permissions.query({ name: 'camera' as PermissionName })).state;
+  } catch (e) {
+    console.warn('permissions API is not supported by browser', e);
```

Comment:
don't warn here, we expect the scenario where it fails and return an appropriate value. You can `console.info` this detail if you feel it adds value to know but we don't want to give developers warnings when they use the function in an intended way

Reply:
I think they should get some information so I will do a console.info()